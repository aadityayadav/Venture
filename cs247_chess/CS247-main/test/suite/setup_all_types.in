setup
+ K a1
+ N b1
+ P c2
+ R d1
+ B e1
+ Q f1
+ k a8
+ n b8
+ p c7
+ r d8
+ b e8
+ q f8
done
