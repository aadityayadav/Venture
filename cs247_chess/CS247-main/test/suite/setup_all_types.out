kn_rbq__
__p_____
________
________
________
________
__P_____
KN_RBQ__
